GitUI/Strings.cs

```diff
@@ -98,10 +95,13 @@ internal sealed class Strings : Translate
 
         private readonly TranslationString _rotInactive = new TranslationString("[ Inactive ]");
 
-        private readonly TranslationString _argumentsText = new TranslationString("Arguments");
-        private readonly TranslationString _commandText = new TranslationString("Command");
-        private readonly TranslationString _workingDirectoryText = new TranslationString("Working directory");
-        private readonly TranslationString _reportBugText = new TranslationString("If you think this was caused by Git Extensions, you can report a bug for the team to investigate.");
+        private readonly TranslationString _userExternalOperationFailed = new TranslationString("User-defined external operation failed");
+        private readonly TranslationString _externalOperationFailed = new TranslationString("External operation failed");
```

Comment:
Most users will likely find this confusing - "user-defined" and "external" errors is something we understand, but users likely wouldn't know a difference - _the app failed, omg, help!_
That was the reason to word the original messages as I did.

Reply:
OK